kedro/extras/datasets/networkx/__init__.py

```diff
@@ -27,11 +27,17 @@
 # limitations under the License.
 
 """``AbstractDataSet`` implementation to save and load NetworkX graphs in JSON
-format using ``NetworkX``."""
+, GraphML and GML formats using ``NetworkX``."""
 
-__all__ = ["NetworkXDataSet"]
+__all__ = ["JSONDataSet", "GMLDataSet", "GraphMLDataSet"]
 
 from contextlib import suppress
 
 with suppress(ImportError):
-    from .networkx_dataset import NetworkXDataSet  # NOQA
+    from .json_dataset import JSONDataSet  # NOQA
+
+with suppress(ImportError):
+    from .gml_dataset import GMLDataSet  # NOQA
+
+with suppress(ImportError):
+    from .graphml_dataset import GraphMLDataSet  # NOQA
```

Comment:
Please could you update this as per the open PR https://github.com/quantumblacklabs/kedro/pull/874 to remove the `# NOQA`?